ocaml-lsp-server/src/progress.mli

```diff
@@ -0,0 +1,17 @@
+open Import
+
+type t
+
+val create :
+     ClientCapabilities.t
+  -> (Server_notification.Progress.t ProgressParams.t -> unit Fiber.t)
+  -> (WorkDoneProgressCreateParams.t -> unit Fiber.t)
```

Comment:
I would maybe add labeled arguments. It's not immediately clear from the signature what these functions do (for me at least)

Reply:
done